doc/source/serve/deployment-graph.md

```diff
@@ -293,15 +293,44 @@ Support control flow in plain python code can be very useful to build dynamic di
 
 Now we've built the entire serve DAG with the topology, args binding and user input. It's time to add the last piece for serve -- a Driver deployment to expose and configure http. We can configure it to start with two replicas in case the ingress of deployment becomes bottleneck of the DAG.
 
-```{note}
-We expect each DAG has a driver class implementation as root, similar to the example below. This is where HTTP ingress are configured and implemented. We provide a default `DAGDriver` to handle simple HTTP parsing, but in this example we put up a custom implementation. 
+Serve provides a default DAGDriver implementation that accepts HTTP request and orchestrate the deployment graph execution. You can import it from `from ray.serve.drivers import DAGDriver`.
+
```

Comment:
I think we should also mention user has all the freedom they want to provide their own DAGDriver implementation as ingress just as if what they would do today, and what we did here is to facilitate holding dag instance by providing a default template. 

Realistically i think there'll be decent # of users who start with our default implementation then add their own code to it rather than our DAGDriver covers everything they need ?

Reply:
it's in two paragraph below.